_targets/build.project.armv7a9-zynq7000

```diff
@@ -19,7 +19,13 @@ export PSH_DEFUSRPWDHASH="0B1ANiYi45IhxkfmUW155/GBd4IRE="
 SIZE_PAGE=$((0x1000))
 PAGE_MASK=$((~(SIZE_PAGE - 1)))
 KERNEL_OFFS=$((0x11000))
-FS_OFFS=$((0x100000))
+FS_OFFS=$((0x800000))
+BS_OFFS=$((0x400000))
```

Comment:
Not sure about the offsets - the proposed change would limit rootfs size to 8MB. Maybe set BS_OFFS to 1MB and determine FS_OFFS based on BS_SZ?

Reply:
I wanted to reserve 4 MB for plo, kernel and syspage stuff. There is no limit for rootfs partion - we just need to fix SPI memory handling to use more than 16 MB. BS_SZ isn't 4 MB - it's almost 4 MB, but not quite. And it should always be the same size. I wan't to reserve 4 MB for it in all cases (e.g. we might want to add bitstream post factum).

Also, I didn't change rootfs size - it was 8 MB before, it is 8 MB now.

Reply:
I wanted to reserve 4 MB for plo, kernel and syspage stuff. There is no limit for rootfs partion - we just need to fix SPI memory handling to use more than 16 MB. BS_SZ isn't 4 MB - it's almost 4 MB, but not quite. And it should always be the same size. I wan't to reserve 4 MB for it in all cases (e.g. we might want to add bitstream post factum).

Also, I didn't change rootfs size - it was 8 MB before, it is 8 MB now.

Reply:
Ok, it's fine if we always want to reserve 4MB for the bitstream.